utils/parametric/_library_client.py

```diff
@@ -350,6 +353,23 @@ def http_client_request(self, method: str, url: str, headers: List[Tuple[str, st
         ).json()
         return resp
 
+    def get_tracer_config(self) -> dict:
+        resp = self._session.get(self._url("/trace/config")).json()
+        return {
+            "dd_service": resp["dd_service"],
+            "dd_log_level": resp["dd_log_level"],
+            "dd_trace_sample_rate": resp["dd_trace_sample_rate"],
+            "dd_trace_enabled": resp["dd_trace_enabled"],
+            "dd_runtime_metrics_enabled": resp["dd_runtime_metrics_enabled"],
+            "dd_tags": resp["dd_tags"],
+            "dd_trace_propagation_style": resp["dd_trace_propagation_style"],
+            "dd_trace_debug": resp["dd_trace_debug"],
+            "dd_trace_otel_enabled": resp["dd_trace_otel_enabled"],
+            "dd_trace_sample_ignore_parent": resp["dd_trace_sample_ignore_parent"],
```

Comment:
If this key does not exist in the response this test will raise a key error. Is this acceptable? Do we want to return a default value? 

Ex:
```suggestion
            "dd_trace_sample_ignore_parent": resp.get("dd_trace_sample_ignore_parent", None),
```